src/mon/MonOpRequest.h

```diff
@@ -22,6 +22,14 @@
 #include "mon/Session.h"
 #include "msg/Message.h"
 
+#include <boost/scoped_ptr.hpp>
+
+struct MonOpInfo {
+
+  MonOpInfo() { }
+  virtual ~MonOpInfo() { }
+};
+
```

Comment:
This is a stub. While it doesn't do anything now, I hope to figure out what it should hold to generically support other classes than `OpInfoDestroyOSD` for the upcoming `osd purge` command.

I could have gotten rid of it and instead have a `void *` in `MonOpRequest`, but think not only type safety but I'm assuming a void pointer will not play well with the `scoped_ptr`. Maybe I'm wrong, in which case I'd be delighted to be shown the path.